include/s3select_functions.h

```diff
@@ -1451,27 +1451,27 @@ bool base_statement::is_function()
   }
 }
 
-bool base_statement::is_aggregate_exist_in_expression(base_statement* e) //TODO obsolete ?
+bool base_statement::is_aggregate_exist_in_expression() const //TODO obsolete ?
```

Comment:
indeed, 
it better to change this interface, to avoid conflicts.
(this specific recursion function, should probably remove)